src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -393,8 +394,21 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                 $vars[$var->getAttribute('key')] = $var->getAttribute('value', '');
             }
 
+            $jwtExpiry = (int) System::getEnv('_APP_FUNCTIONS_BUILD_TIMEOUT', 900);
+            $jwtObj = new JWT(System::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', $jwtExpiry, 0);
+            $apiKey = $jwtObj->encode([
+                'projectId' => $project->getId(),
+                'scopes' => $function->getAttribute('scopes', [])
+            ]);
+
+            $protocol = System::getEnv('_APP_OPTIONS_FORCE_HTTPS') == 'disabled' ? 'http' : 'https';
+            $hostname = System::getEnv('_APP_DOMAIN');
+            $endpoint = $protocol . '://' . $hostname . "/v1";
```

Comment:
this feels unnecessary. lets remove it

Reply:
This is to provide APPWRITE_FUNCTION_API_ENDPOINT, it's copy&paste, we do same logic in other places where we want to provide APPWRITE_FUNCTION_API_ENDPOINT